plug.py

```diff
@@ -401,6 +401,32 @@ def notify_capidata(
     return error
 
 
+def notify_capi_fleetcarrierdata(
+    data: companion.CAPIData,
+    is_beta: bool
+) -> Optional[str]:
+    """
+    Send the latest CAPI Fleetcarrier data from the FD servers to each plugin.
+
+    :param data:
+    :param is_beta: whether the player is in a Beta universe.
+    :returns: Error message from the first plugin that returns one (if any)
+    """
+    error = None
+    for plugin in PLUGINS:
+        fc_data = plugin._get_func('capi_fleetcarrier')
+
+        if fc_data:
+            try:
+                newerror = fc_data(data, is_beta)
```

Comment:
Yes, they're both still things, and we ought to put that `callable(...)` check on all the plugin calling code in plug.py.

FYI @aussig the deep copy is so that one plugin mutating the data doesn't affect a subsequent one.

Reply:
Will implement both.